File: app/src/app/features/food/resources/nutrition/nutrition-value.resource.ts
import { UnitResource } from 'src/app/features/settings/resources/unit/unit.resource';

export class NutritionValueResource {
  totalCalories: number;
  totalCarbohydrates: number;
  totalFat: number;
  totalProtein: number
}

export class NutritionValueModel {
  id: number;
  baseQuantity: number;
  UnitId: number;
  calories: number;
  fat: number;
  protein: number;
  carbohydrates: number;
  unit: UnitResource;
}

File: api/BeHealth.Business/Resources/FoodManamgnet/Nutrition/NutritionValueResource.cs
using BeHealth.Business.Resources.Settings;
using System;
using System.Collections.Generic;
using System.Text;

namespace BeHealth.Business.Resources.FoodManamgnet.Nutrition
{
    public class NutritionValueResource
    {
        public long ID { get; set; }

        public double Fat { get; set; }

        public double Carbohydrates { get; set; }

        public double Protein { get; set; }

        public double Calories { get; set; }

        public double BaseQuantity { get; set; }

        public UnitResource Unit { get; set; }
    }
}
